content/index.md

```diff
@@ -10,6 +10,7 @@ layout: overview
   </h1>
   <p class="intro">Jump right into the API documentation to get started.</p>
   <a href="millennium/dstu2/" class="button">Browse the Millennium documentation</a>
+  <a href="soarian/dstu2/" class="button">Browse the Soarian documentation</a>
```

Comment:
Having buttons of different sizes next to each other looks very strange to me.  Do we want to add css to keep the buttons the same size as each other?